spec/models/manageiq/providers/ansible_tower/automation_manager/credential_spec.rb

```diff
@@ -1,5 +1,9 @@
 require 'support/ansible_shared/automation_manager/credential'
 
-describe ManageIQ::Providers::AnsibleTower::AutomationManager::Credential do
+describe ManageIQ::Providers::AnsibleTower::AutomationManager::ScmCredential do
```

Comment:
Because `ManageIQ::Providers::AnsibleTower::AutomationManager::Credential` is not a concrete class and doesn't have `TOWER_KIND`.

Maybe I'll rename the file to `scm_credential_spec.rb`?